### ISSUE TITLE
Inappropriate ioctl for device on windows 10. 

### ISSUE THREAD

**Issue by hanjoosten** [ndmitchell/ghcid#301]
When I start ghcid from within vscode (command `start ghcid`), the new terminals starts with the following output:

~~~
Loading stack ghci --test --bench --no-load --ghci-options=-fno-code --ghci-options=-fno-break-on-exception --ghci-options=-fno-break-on-error --ghci-options=-v1 --ghci-options=-ferror-spans --ghci-options=-j ...
stty: standaardinvoer: Inappropriate ioctl for device
?[0mampersand> configure (lib + exe + test)?[0m
?[0mConfiguring ampersand-4.0.0...?[0m
~~~

I am on a windows 10 machine. Nothing fancy as far as I know, but I get the message `Inappropriate ioctl for device`. I wonder if that is caused by my localization. I use a Dutch version of windows. 

After I modify something in the code, ghcid gets triggered for another load cycle. The output then becomes normal ansi. Errors are colored fine. 

This isn't a big deal, but I still thought it would be good to raise an issue.

**Comment by hanjoosten** [comment]
For what it is worth: stty is a valid command, on the command line it returns:

~~~
C:\Users\hjo20125>stty
speed 38400 baud; line = 0;
-imaxbel iutf8
-echoe -echok -echoctl -echoke
~~~

**Comment by hanjoosten** [comment]
Could be very well related with https://github.com/commercialhaskell/stack/issues/5038

**Comment by ndmitchell** [comment]
Yep, it pretty much certainly is stack. Thanks for the report, I'll leave this to track the upstream issue.
